cfme/infrastructure/cluster.py

```diff
@@ -20,7 +22,7 @@
 
 
 def nav_to_cluster_through_provider(context):
-    sel.force_navigate('infrastructure_provider', context=context)
```

Comment:
Can we add a TODO on this one so it doens't get missed it will need replacing with it's own NavigateStep